katcp/resource_client.py

```diff
@@ -391,8 +391,22 @@ def list_sensors(self, filter="", strategy=False, status="",
             dict.items(self.sensor), filter, strategy, status, use_python_identifiers, tuple, refresh)
 
     @tornado.gen.coroutine
-    @steal_docstring_from(resource.KATCPResource.set_sensor_strategy)
```

Comment:
I would prefer it if you kept/added the abstract method definitions in resource.py. The idea is that someone could implement another implementation (e.g. simulated) of a KATCP resource by basing their code on the ABC, with the ABC defining the minimum expected API that should be exposed by a KATCPResource compatible implementor.